app/src/main/res/values-nl/strings.xml

```diff
@@ -56,7 +58,7 @@
     <string name="action_restart_tutorial">Herstart Uitleg</string>
 
     <!-- snackbar -->
-    <string name="snackbar_description">Weet u zeker dat u het wilt verwijderen?</string>
+    <string name="snackbar_description">Herinneringen verwijderd.</string>
```

Comment:
verwijdert met t. Het is de tegenwoordige tijd.

Reply:
het is wel verleden tijd. De herinneringen zijn al verwijderd wanneer het zichtbaar word.